service/controller/app/v1/resource/configmap/create.go

```diff
@@ -28,6 +32,24 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		_, err = cc.K8sClient.CoreV1().ConfigMaps(configMap.Namespace).Create(configMap)
 		if apierrors.IsAlreadyExists(err) {
 			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("already created configmap %#q in namespace %#q", configMap.Name, configMap.Namespace))
+		} else if apierrors.IsTimeout(err) {
```

Comment:
`tenant.IsAPINotAvailable` also checks for certain timeout errors. Why do we need to explicitly use `apierrors.IsTimeout` here now?

Reply:
I didn't know that `apierrors` check on it, since I saw the best practice on here https://github.com/giantswarm/cluster-operator/blob/4161b4dfdcb02b73c4574d1c6f9335ac0f514587/pkg/v17/resource/namespace/current.go#L57-L75

Did you add timeoutError later then? Should I update cluster-operator?

Reply:
@rossf7 > Do you prefer to delete Timeout err check in general, since `tenant.IsAPINotAvailable` covers all the cases in here. (I can do update cluster-operator after this)

Reply:
Sorry, I'm late here and I added this when trying to fix the persistent connection problems we get in `viking`.

I think Tim is right here and we should just use `tenant.IsAPINotAvailable`. Yes please also remove from the WIP version of cluster-operator. <3